### PR TITLE
Adding context handler to graph manager and sc command line test.

### PR DESCRIPTION
**Pull request by charlesvardeman**
Added a JSON-LD context handler in graphRegistry to merge the context dictionaries. Added a quick and dirty test to make sure the sc command doesn't get broken by setup.py install version dependencies and requirements.

### CODE REVIEW

File: sc/graphRegistry.py [hunk 1]
```diff
@@ -2,55 +2,150 @@
...
+        """
+        if existing_graph:
+            self.global_graph.parse(data=existing_graph, format='turtle')
 
     @classmethod
     def get_registry(self):
+        """get_registry: Returns the dictionary of all registered vocbularies.
+
+        All of the vocabularies in the graphRegistry store their
+        their instances in the REGISTRY dictionary. This provides an access
```
Comment (ghost) [inline sc/graphRegistry.py:49]: Minor: double use of 'their'.

File: sc/graphRegistry.py [hunk 2]
```diff
@@ -2,55 +2,150 @@
...
+
+        Add an existing graph object to the default global graph using a
+        turtle serialzation to parse the graph.
+
+        Args:
+            existing_graph (str):    Turtle serialized rdf graph.
+
+        """
+        self.global_graph.parse(data=existing_graph, format='turtle')
         pass
```
Comment (ghost) [inline sc/graphRegistry.py:149]: Should the `pass` line be removed now?

File: tests/unit/test_cmdline.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""Tests for SC command line execution.
+
+This module tests that the sc command as setup from setup.py
+executes correctly and returns the help useage. This is meant to
+to be a quick and dirty test to make sure that there isn't a depenency
```
Comment (ghost) [inline tests/unit/test_cmdline.py:6]: Double 'to'. Also, 'depenency' typo.

File: sc/graphRegistry.py (hunk 1 above)
Comment (charlesvardeman) [inline sc/graphRegistry.py:49]: Fixed typo for next pull request.

File: tests/unit/test_cmdline.py (hunk 3 above)
Comment (charlesvardeman) [inline tests/unit/test_cmdline.py:6]: Fixed typo in my repo. Will do a pull on the typo fix.

File: sc/graphRegistry.py (hunk 2 above)
Comment (charlesvardeman) [inline sc/graphRegistry.py:149]: Pass should be removed, but I'm in the process of wiring this code up to scMetadata. I'll fix that when I add that code.

**Pull request merged**